javascript1/week4/homework.md

```diff
@@ -61,7 +61,7 @@ Go over your homework one last time:
 
 - [ ] Does every file run without errors and with the correct results?
 - [ ] Have you used `const` and `let` and avoided `var`?
-- [ ] Do the variable, function and argument names you created follow the [Naming Conventions](https://github.com/HackYourFuture/fundamentals/blob/main/fundamentals/naming_conventions.md)?
+- [ ] Do the variable, function and argument names you created follow the [Naming Conventions](https://github.com/HackYourFuture/fundamentals/blob/master/fundamentals/naming_conventions.md)?
```

Comment:
Good catch. I guess this could have been introduced with my PR changing all `master` instances to `main`. Thought I tested it for 404s.

Reply:
@adamblanchard just saw another place. Could you update it, too - https://github.com/HackYourFuture-CPH/JavaScript/commit/73d286d25d07a79e2e3a28a47255dfc60a9dce34#r144164214

Reply:
Thanks @jason-vasilev!